README.rst

```diff
@@ -1,4 +1,4 @@
-Visualization Interface Tool
+PyAdditive-Widgets
```

Comment:
I think you got the title wrong 😅

Reply:
@AlejandroFernandezLuces That's what happens when I jump around! Sorry!